main.nf

```diff
@@ -122,14 +130,17 @@ if (!params.fastq) {
         output:
             file 'fastq.fq.gz' into fastq
 
-        publishDir 'results'
-
         module 'bioinfo-tools'
         module "$params.modules.samtools"
 
         // We only need one core for this part
-        clusterOptions = {
-            "-A $params.project -p $params.runspecs.core $params.runspecs.extra"
+        if ( nextflow_running_as_slurmjob() ) {
```

Comment:
Well, it's different processes, it might be possible to put directives in subroutines, but I'm not sure, I will test that and get back to you.